setup.cfg

```diff
@@ -1,3 +1,52 @@
+[metadata]
+name = lomap2
+description = Lead Optimization Mapper 2
+long_description = file: README.md
+long_description_type = text/markdown
+author = Gaetano Calabro and David Mobley
+author_email = 
```

Comment:
leaving this blank; previously had `gcalabro -at- uci.edu`: I think that this are supposed to actually be clickable for PyPI, and I wouldn't be sure that the UCI email is still alive.